fontc/src/main.rs

```diff
@@ -1459,4 +1514,46 @@ mod tests {
             cmap4.id_delta()
         );
     }
+
+    #[test]
+    fn hmtx_of_one() {
+        let temp_dir = tempdir().unwrap();
+        let build_dir = temp_dir.path();
+        let result = compile(test_args(build_dir, "glyphs2/OneGlyph.glyphs"));
+
+        let raw_hmtx = result.be_context.get_hmtx();
+        let hmtx = Hmtx::read_with_args(FontData::new(&raw_hmtx.buf), &(1, 1)).unwrap();
```

Comment:
what's `&(1, 1)`, I suppose something from hhea needed to parse hmtx.. I wish rust had keyword args sometimes

Reply:
I agree, and the docs for read with args for hmtx are pretty obtuse too. @cmyr thoughts?

Reply:
we have https://github.com/googlefonts/fontations/issues/108 open for this; I did not expect this method to really ever be called directly from user code. I'll hack on this now, haven't written code in a while and this feels like a good warmup.

Reply:
I will happily update once we have something in fontations but I would prefer not to block this PR on this.